wordpress-mobile/WordPress-iOS
# Minor improvements to Siri Shortcuts experience

### Description 
Fixes #10544.

### Testing

- Checkout the branch and verify that existing test pass.
- For context, it may be helpful to refer to these [prior instructions](https://github.com/wordpress-mobile/WordPress-iOS/pull/10225)
- Navigate to _Me : App Settings_ & confirm that the new settings appear as shown below.
- Verify that the new button works.

### Screenshot(s)
![pr](https://user-images.githubusercontent.com/221062/49059026-ad17da00-f1bb-11e8-8b47-4d57c915b546.png)
